content/security/for-admins/scim.md

```diff
@@ -58,3 +61,200 @@ You must make sure you have [configured SSO](single-sign-on/configure/_index.md)
 {{< /tab >}}
 {{< /tabs >}}
 
+## Enable SCIM in your IdP
+
+{{< tabs >}}
+{{< tab name="Okta" >}}
+
+### Enable SCIM
+
+1. Go to the Okta admin portal.
+2. Go to the app you created when you configured your SSO connection.
+3. On the app page, go to the **General** tab and select **Edit App Settings**.
+4. Enable SCIM provisioning, then select **Save**.
+5. Now you can access the **Provisioning** tab. Navigate to this tab, then select **Edit SCIM Connection**.
+6. To configure SCIM in Okta, set up your connection like the following:
+    - SCIM Base URL: SCIM connector base URL (copied from Docker Hub)
+    - Unique identifier field for users: `email`
+    - Supported provisioning actions: **Push New Users** and **Push Profile Updates**
+    - Authentication Mode: HTTP Header
+    - SCIM Bearer Token: HTTP Header Authorization Bearer Token (copied from Docker Hub)
+7. Select **Test Connector Configuration**.
+8. Review the test results.
+9. Select **Save**.
+
+### Enable synchronization
+
+1. Go to **Provisioning > To App > Edit**.
+2. Enable **Create Users**, **Update User Attributes**, and **Deactivate Users**.
+3. Select **Save**.
+4. Remove unnecessary mappings. The necessary mappings are:
+    - Username
+    - Given name
+    - Family name
+    - Email
+
+{{< /tab >}}
+{{< tab name="Entra ID SAML 2.0" >}}
+
+1. In the Azure admin portal, go to **Enterprise Applications**, then select the **Docker** application you created when you set up your SSO connection.
+2. Go to **Provisioning** and select **Get Started**.
+3. Select **Automatic** provisioning mode.
+4. Enter the SCIM Base URL and API Token from Docker Hub into the **Admin Credentials** form.
+5. Test the connection, then select **Save**.
+6. Go to  **Mappings** , then select **Provision Azure Active Directory Groups**.
+7. Set the **Enabled** value to **No**.
+8. Select **Provision Azure Active Directory Users**.
+9. Remove all unsupported attributes.
+10. Select **Save**.
+11. Set the provisioning status to **On**.
+
+{{< /tab >}}
+{{< tab name="OneLogin" >}}
+
+1. Go to the OneLogin admin portal.
+2. Go to the app that you created when you configured SSO.
+3. Go to the **Access** tab and enable the app for the `ol_role` that you created when you configured SSO.
```

Comment:
`ol_role` was a reference used within an internal doc, I think in this separated context has no meaning, i'd suggest to remove it and just make reference to a role previously created during SSO setup.

Reply:
Removing OneLogin tab.